Framework/Algorithms/test/MayersSampleCorrectionStrategyTest.h

```diff
@@ -50,92 +48,120 @@ class MayersSampleCorrectionStrategyTest : public CxxTest::TestSuite {
     using std::sqrt;
     const size_t nypts(100);
     std::vector<double> signal(nypts, 2.0), tof(nypts), error(nypts);
+
     std::transform(signal.begin(), signal.end(), error.begin(),
                    (double (*)(double))sqrt);
     std::generate(tof.begin(), tof.end(), Incrementer(100.0));
-    MayersSampleCorrectionStrategy mscat(createTestParameters(), tof, signal,
-                                         error);
+
+    Points tofPoints(tof);
+    Histogram histo(tofPoints, Counts(signal), CountStandardDeviations(error));
+
+    MayersSampleCorrectionStrategy mscat(createTestParameters(), histo);
 
     // Correct it
-    mscat.apply(signal, error);
+
+    auto outHisto = mscat.apply(histo);
+
+    const auto &tofVals = outHisto.x();
+    const auto &signalVals = outHisto.y();
+    const auto &errVals = outHisto.e();
 
     // Check some values
     const double delta(1e-06);
-    TS_ASSERT_DELTA(100.0, tof.front(), delta);
-    TS_ASSERT_DELTA(199.0, tof.back(), delta);
+    TS_ASSERT_DELTA(100.0, tofVals.front(), delta);
+    TS_ASSERT_DELTA(199.0, tofVals.back(), delta);
 
-    TS_ASSERT_DELTA(0.37497317, signal.front(), delta);
-    TS_ASSERT_DELTA(0.37629282, signal.back(), delta);
+    TS_ASSERT_DELTA(0.37497317, signalVals.front(), delta);
+    TS_ASSERT_DELTA(0.37629282, signalVals.back(), delta);
 
-    TS_ASSERT_DELTA(0.26514607, error.front(), delta);
-    TS_ASSERT_DELTA(0.2660792, error.back(), delta);
+    TS_ASSERT_DELTA(0.26514607, errVals.front(), delta);
+    TS_ASSERT_DELTA(0.2660792, errVals.back(), delta);
   }
 
   void
   test_Corrects_Both_Absorption_And_Multiple_Scattering_For_Histogram_Data() {
     using std::sqrt;
     const size_t nypts(100);
-    std::vector<double> signal(nypts, 2.0), tof(nypts + 1), error(nypts);
+    std::vector<double> signal(nypts, 2.0), tof(nypts), error(nypts);
```

Comment:
Why was this changed? According to the test name this should be testing for histogram data, i.e., bin edges and not points.